MonkeyTrap/app/src/main/java/com/novoda/monkeytrap/OverlayView.java

```diff
@@ -0,0 +1,45 @@
+package com.novoda.monkeytrap;
+
+import static android.graphics.PixelFormat.TRANSLUCENT;
+import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;
+import static android.view.WindowManager.LayoutParams.FLAG_LAYOUT_INSET_DECOR;
+import static android.view.WindowManager.LayoutParams.FLAG_LAYOUT_IN_SCREEN;
+import static android.view.WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS;
+import static android.view.WindowManager.LayoutParams.FLAG_NOT_FOCUSABLE;
+import static android.view.WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL;
+import static android.view.WindowManager.LayoutParams.TYPE_SYSTEM_ERROR;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.view.Gravity;
+import android.view.WindowManager;
+import android.widget.FrameLayout;
+
+class OverlayView extends FrameLayout {
+
+    private static final int SAFETY_MARGIN = 20;
+
+    static WindowManager.LayoutParams createLayoutParams(Resources resources) {
+        final WindowManager.LayoutParams params =
+                new WindowManager.LayoutParams(MATCH_PARENT, retrieveStatusBarHeight(resources) + SAFETY_MARGIN,
```

Comment:
for safety :trollface: 

I imagine it is because the touch are is slightly larger than the visible area of the notifications bar. I took it from https://github.com/Orange-OpenSource/simiasque (as the README says)